contrib/chart/backstage/values.yaml

```diff
@@ -35,6 +35,25 @@ backend:
     limits:
       memory: 1024Mi
 
+  # `existingSecret` is a reference to an existing secret containing the sensitive information for the app-config. Make sure the secrets define the required information. EXAMPLE: 
```

Comment:
Should the example text for the secret here contain some dummy values or at least make a note that some of the values, if they are missing, they can make backstage fail to start? (at least IIRC that was the case for some)